planemo/commands/cmd_list_repos.py

```diff
@@ -0,0 +1,35 @@
+"""Module describing the planemo ``ci_find_repos`` command."""
+from __future__ import print_function
+
+import click
+
+from planemo import options
+from planemo.ci import print_as_yaml
+from planemo.cli import command_function
+from planemo.shed import (
+    _realize_effective_repositories,
+    find_raw_repositories,
+)
+
+
+@click.command('ci_list_repos')
```

Comment:
This is confusing - this has nothing to do with CI right? Why prefixed this way and why was print_as_yaml put in there? The ci module has to do with like diffing Travis branches and stuff doesn't it?

Reply:
Yep, forgot to push a commit that fixes that ...